### DISCUSSION TITLE
Virtual Worlds and Simulations meeting agenda for 2023-02-08

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-virtualworldsim#2] [Meetings]
## Meeting Details

- **Date/Time:** February 8th, 2023 @ Wednesday 12:00 PM ET / 17:00 UTC / 6:00 PM CET
- **Location:** Link should be posted in the `#Virtual-world-sim` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** TBA
- **Note Taker** Person

The [FIG Virtual Worlds and Simulations Meetings](https://github.com/Open-MV/fig-virtualworldsim/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda
First meeting and operating
FIG charter discussion
Database Feature triage

**Discuss agenda from proposed topics**

## Outcomes from Discussion topics

USD specification world - authoring format - Major companies have done some specifications for physics
Need to look at rigid body simulation -  and distributed physics systems - objects and sandboxed virtual machines

DIscussed how entities are defined and minimal bridge elements -
Open container format and Open object formats
File and block of data - metaverse can rely on the base - to move objects around

## Action Items

**Create actionable items from proposed topics**

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by LaylBongers** [comment]
A few suggested topics:
- object format, container, "file-as-object, "self-descriptive object"
- protocol middleware (QUIC? also discussed in networking)
- interfacing with third party virtual worlds
- prototype object viewer
- prototype client/server
